ui/src/utils/prop-types.js

```diff
@@ -359,6 +359,18 @@ export const secrets = arrayOf(
   })
 );
 
+export const notificationType = oneOf([
+  'EMAIL',
+  'PULSE',
+  'IRC_CHANNEL',
+  'IRC_USER',
+]);
```

Comment:
In `src/utils/contants.js` create a variable `DENYLIST_NOTIFICATION_TYPES` and refer to it instead of hard-coding the values here.

```jsx
export const DENYLIST_NOTIFICATION_TYPES = {
  'EMAIL': 'EMAIL',
  // ...
};
```